README.md

```diff
@@ -11,3 +11,154 @@ This Repository
 ===============
 
 This repository contains the markup needed to deploy the floeproject.org website.
+
+
+News Item Template
```

Comment:
@dayotte The readme should also have instructions on updating the news/index.html and the floe front page. Perhaps a general description of these activities is enough? i.e.
- "Also remember to update the front page (i.e. index.html) with a link to your new News post.
- "Also, don't forget to update the news/index.html page too!"
etc.